src/initialize.js

```diff
@@ -290,7 +290,7 @@ export async function initialize({
   analyticsService = SegmentAnalyticsService,
   authService = AxiosJwtAuthService,
   authMiddleware = [],
-  externalScripts = [GoogleAnalyticsLoader],
+  externalScripts = [GoogleAnalyticsLoader, GoogleTagManagerLoader],
```

Comment:
Since we have `GoogleAnalyticsLoader` defined here already, it looks like the right place to add support for something like GTM.  However, adding support for arbitrary external scripts, however popular, is not really maintainable for the project.

Instead, it would be better if we made `externalScripts` configurable via `env.config.jsx`, removing Google Analytics support in the process but allowing operators to add any external script they need without requiring the platform to explicitly support it.  Is this a change you're comfortable contributing?

Reply:
Hello @arbrandes, thank you for the idea of making `externalScripts` configurable!

We'll discuss this approach with our team and may consider creating a separate PR to explore that direction.

For now, I think to keep the current implementation as-is, since in our projects we frequently use GTM in this way. We’d also like to gather more feedback from the community, including real-world examples of GTM usage, to help refine and potentially improve our solution.

Reply:
@arbrandes thanks for your feedback! Can we go ahead with the changes we have in this PR and create a separate issue to implement the functionality of adding `externalScripts` via `env.config.jsx`?

Reply:
Thanks for context!